### PR TITLE
Created a slack.yml file

### PR DESCRIPTION
**Pull request by debjani1413**
Signed-off-by: debjani1413 <deydebjani916@gmail.com>

Description

The slack.yml file was missing in the workflow of the .github folder. Now it has been successfully added.

### CODE REVIEW

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Slack Notify on Star
+on: watch
+jobs:
+  star-notify:
+    name: Notify Slack on star
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get current star count
+      run: |
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/meshery/meshery' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
```
Comment (leecalcote) [inline .github/workflows/slack.yml:10]:
```suggestion
        echo "STARS=$(curl --silent 'https://api.github.com/repos/meshery/meshery-adapter-template' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
```

File: .github/workflows/slack.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    - name: Notify slack
+      env:
+        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      uses: pullreminders/slack-action@master
+      with:
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred Meshery! (https://github.com/meshery/meshery/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```
Comment (leecalcote) [inline .github/workflows/slack.yml]:
```suggestion
        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred Meshery! (https://github.com/meshery/meshery-adapter-template/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```

**Comment by debjani1413** [comment]
@leecalcote I did the required changes do let me know!!

**Review by leecalcote — APPROVED** [review]
👍

**Pull request merged**